### ISSUE TITLE
Seed Badger/FRAXBP pool from Treasury

### ISSUE THREAD

**Issue by dapp-whisperer** [Badger-Finance/badger-strategies#78]
[Context](https://github.com/Badger-Finance/Badger-Project-Planning/issues/101)
Depends on: https://github.com/Badger-Finance/badger-strategies/issues/77

**Comment by gosuto-inzasheru** [comment]
@dapp-whisperer how can we move this issue to the repo? i dont see the migration link, not sure if that is because i am just missing perms or something else..?

**Comment by spadaboom** [comment]
Also, as part of this we should be deploying a gauge and creating a gauge vote which requires an wallet with 2500 veCRV to execute. 

Liquidity seeding doesn't need to be exceptionally large. Maybe $5k for now before growing that liquidity deployment when gauges are approved. 

![Image](https://user-images.githubusercontent.com/69224784/190662308-0c9d4311-7d9a-4d66-a3a6-2caa63e7162a.png)

**Comment by gosuto-inzasheru** [comment]
can be closed, see https://github.com/Badger-Finance/badger-multisig/issues/825#issuecomment-1259256415

**Comment by FreddytheFilosopher** [comment]
Before closing two more actions are required:

1. Please "Enact" the gauge on Curve: https://dao.curve.fi/vote/ownership/215
2. Call addPool(guage address) on the Convex pool manager: https://discord.com/channels/820795644494610432/820795644494610435/1025408870098669648

**Comment by gosuto-inzasheru** [comment]
pool has been enacted, and i was able to `addPool` on the manager permissionlessly: https://etherscan.io/tx/0x40aa3928155784977e5a195388b2933a814adf6b8f707dfec01d009c24007e71

**Comment by gosuto-inzasheru** [comment]
please close and if necessary create new ticket, scope here is too narrow for more actions

**Comment by gosuto-inzasheru** [comment]
```
>>> booster = Contract('0xF403C135812408BFbE8713b5A23a04b3D48AAE31')
>>> booster.poolInfo(123)
("0x09b2E090531228d1b8E3d948C73b990Cb6e60720", "0x25f0b7c3A7A43b409634a5759526560cC3313d75", "0x455279344f84a496615DC0ffa0511D2E19ec19d8", "0xe0705A91984b076C250d410A41f615380aF1C2ed", "0x7Da6663D46Ad439ec0b2d327c23052Df580950AD", False)
```
